modules/core/src/main/java/org/apache/ignite/internal/processors/datastructures/GridCacheCountDownLatchImpl.java

```diff
@@ -249,20 +252,16 @@ private void initializeLatch() throws IgniteCheckedException {
                 internalLatch = CU.outTx(
                     retryTopologySafe(new Callable<CountDownLatch>() {
                         @Override public CountDownLatch call() throws Exception {
-                            try (IgniteInternalTx tx = CU.txStartInternal(ctx, latchView, PESSIMISTIC, REPEATABLE_READ)) {
```

Comment:
You must leave this code as-is because a latch with the same name can be being initialized on several nodes at the same time. This distributed transaction protects from this case.